code/modules/paperwork/fax.dm

```diff
@@ -0,0 +1,133 @@
+/obj/machinery/fax
+	name = "Fax Machine"
+	desc = "Bluespace technologies on the application of bureaucracy."
+	icon = 'icons/obj/bureaucracy.dmi'
+	icon_state = "fax"
+	density = TRUE
+	power_channel = AREA_USAGE_EQUIP
+	max_integrity = 300
```

Comment:
why does it have 3x the normal integrity? It's just a machine

Reply:
> why does it have 3x the normal integrity? It's just a machine

This is the value used in most machines. For example, many consoles have this indicator. At least I did not find any recommendations on this.